src/applications/edu-benefits/10203/pages/personalInformation.js

```diff
@@ -45,13 +55,46 @@ export const uiSchema = {
       },
     },
   },
-};
-
-export const schema = {
-  ...contactInformation.schema,
-  properties: {
-    veteranAddress: address.schema(fullSchema10203, true),
-    'view:otherContactInfo': ciSchemaProperties['view:otherContactInfo'],
-    preferredContactMethod,
+  receiveTexts: {
+    'ui:title':
+      'I would like to receive text messages from VA about my GI Bill benefits.',
+  },
+  'view:housingPaymentInfo': {
+    'ui:description': receiveTextsAlert,
+    'ui:options': {
+      hideIf: data =>
+        !data?.receiveTexts ||
+        phoneNumberFormatted(data['view:otherContactInfo']?.mobilePhone),
+    },
+  },
+  'view:receiveTextsInfo': {
+    'ui:description': receiveTextsNote,
   },
 };
+
+export const schema = environment.isProduction()
```

Comment:
There is a production in schema but not in uiSchema

Reply:
Adding a similar check for UISchema